core/build.gradle.kts

```diff
@@ -3,10 +3,13 @@
  * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENCE file.
  */
 
+import org.jetbrains.dokka.gradle.DokkaTaskPartial
 import org.jetbrains.kotlin.gradle.plugin.*
 
 plugins {
     kotlin("multiplatform")
+    id("org.jetbrains.kotlinx.kover") version "0.7.0"
```

Comment:
Latest version is `0.7.1`

Reply:
Thanks! Also, I'll update other dependencies.